### PR TITLE
Set media element

### PR DESCRIPTION
**Pull request by AxelDelmas**
Pass media element to peer agent

### CODE REVIEW

**Comment by maxbok** [comment]
:+1:
Don't forget to update the README!

**Comment by AxelDelmas** [comment]
Why is the default target branch master on this repo? Is it normal?
I originately meant to merge this branch in dev (it is based on dev)

**Comment by tchakabam** [comment]
@AxelDelmas we have set it master because its public, and as requested by @remijulien from a product pov its better that the first branch people get to when they reach the repo is master

File: lib/hlsjs-p2p-wrapper-private.js [hunk 1]
```diff
@@ -161,6 +161,16 @@ class HlsjsP2PWrapperPrivate {
         return !!this.peerAgentModule;
     }
 
+    _setMediaElement(hlsjs, hlsEventsEnum) {
```
Comment (tchakabam) [inline lib/hlsjs-p2p-wrapper-private.js]: would be great if we could keep this class as documented as i initially built it

Comment (tchakabam) [inline lib/hlsjs-p2p-wrapper-private.js]: why the underscore in the method name? the whole class is a private API.

**Comment by tchakabam** [comment]
👍 weird that these tests fail. they must be flaky. just re-ran the job once.

**Comment by tchakabam** [comment]
Ok then seems something is wrong with the Hls.js version compatibility that is currently pulled in. How come its not broken on dev branch though?

**Comment by tchakabam** [comment]
its broken on dev too. fixing it now ... @AxelDelmas

File: lib/hlsjs-p2p-wrapper-private.js (hunk 1 above)
Comment (AxelDelmas) [inline lib/hlsjs-p2p-wrapper-private.js]: Leading underscore is our convention everywhere for methods / properties that are meant to be private, even if they are not truly private like in ES6.

Comment (tchakabam) [inline lib/hlsjs-p2p-wrapper-private.js]: sure i know. in the wrapper class there are more functions that are not supposed to be called from outside. but anyway lets go like this for now.

**Comment by AxelDelmas** [comment]
Closing, will merge in dev

**Pull request closed without merging**